symfony/plugins/orangehrmMaintenancePlugin/AccessStrategy/AccessStrategy.php

```diff
@@ -115,67 +115,75 @@ public function getMaintenanceService()
     /**
      * @param MaintenanceService $maintenanceService
      */
-    public function setMaintenanceService($maintenanceService)
+    public function setMaintenanceService(MaintenanceService $maintenanceService): void
     {
         $this->maintenanceService = $maintenanceService;
     }
 
     /**
-     * @param $employeeNumber
+     * @param int $employeeNumber
      * @return array
      */
-    public function getMatchByValues($employeeNumber)
+    public function getMatchByValues(int $employeeNumber): array
     {
-        $entityFieldMap = array();
-
+        $entityFieldMap = [];
         $entityFieldMap[$this->getMatchBy()[0]['match']] = $employeeNumber;
-        if ($this->getMatchBy()[0]['join']) {
+
+        if (in_array('join', $this->getMatchBy()[0])) {
             $entityFieldMap['join'] = $this->getMatchBy()[0]['join'];
         }
+
         return $entityFieldMap;
     }
 
     /**
-     * @param $matchByValues
-     * @param $table
-     * @return mixed
-     * @throws DaoException
+     * @param array $matchByValues
+     * @param string $table
+     * @return array|null
      */
-    public function getEntityRecords($matchByValues, $table)
+    public function getEntityRecords(array $matchByValues, string $table): ?array
     {
         return $this->getMaintenanceService()->extractDataFromEmpNumber($matchByValues, $table);
     }
 
     /**
-     * @param $aceessClassName
-     * @param $currentValue
-     * @return mixed
+     * @param string $accessClassName
+     * @param string $currentValue
+     * @return string|null
      */
-    public function getFormattedValue($aceessClassName, $currentValue)
+    public function getFormattedValue($accessClassName, $currentValue)
     {
-        $this->getRealValueClass = new $aceessClassName();
+        $accessClassName='OrangeHRM\Maintenance\FormatValueStrategy'."\\".$accessClassName;
+        $this->getRealValueClass = new  $accessClassName();
         return $this->getRealValueClass->getFormattedValue($currentValue);
     }
 
     /**
      * @param $accessEntity
      * @return array
      */
-    public function addRecordsToArray($accessEntity)
+    public function addRecordsToArray($accessEntity): array
     {
-
         $parameters = $this->getParameters();
-        $data = array();
+        $data = [];
+
         foreach ($parameters as $field) {
-            $columnName = $field['field'];
-            if ($accessEntity->$columnName) {
-                $value = $accessEntity->$columnName;
-                if ($field['class']) {
+            $columnName=$field['field'];
```

Comment:
seems not well formated
```suggestion
            $columnName = $field['field'];
```